meetings/2021-03-22.md

```diff
@@ -0,0 +1,91 @@
+* [keyId's do not exactly refer to keys anymore #151](https://github.com/solid/authentication-panel/issues/151)
+
+## Present
+
+* Henry Story
+* Aaron Coburn
+* Matthieu Bosquet
+* Sarven Capadisli
+* Elf Pavlik
+* Benoit Alessandroni
+
+## Minutes
+
+### Any discussion/follow-ups to WICG/SolidCG meeting
+
+Sarven: They added note to README mentioning name collision.
+...: We should come back to them with what solid would need from web browsers. Create issue?
+
+Henry: Good idea to create an issue. It need to first implement HTTPSig, ACLs and then get to Credentials before I can start constructively looking at how we could work together. So my guess this will take 3 months before  I can give an informed answer.
+
+Pavlik: I would follow their work especially what they do with 'directed identifiers' (site specific ones)
+
+Sarven: scope is identity/authentication but may even edgy stuff; native RDF parsing and serializing?
+
+### [Proposal: change webid claim to solid #146](https://github.com/solid/authentication-panel/issues/146)
+
+Aaron: we have some +1 on the issue. Pavlik suggested using IRIs for claim names and for scopes.
+I generally don't see it in other systems but they can be used.
+Pavlik: if we don't want to use IRI we would need to regsiter that JWT claim
+Aaron: is solid is taking route of registering strings with IANA or using IRIs which we don't have to register
+Sarven: I don't know if we have global position on it, we did it for `acl` which has a history. `describedby` is registered by LDP spec.
+Aaron: In that case, if something we will change to using IRI sounds like linked data friendly approach.
+Sarven: If we think it can be used beyond solid we could consider registering with IANA.
+Henry: Moving from "WebID" to something more general such as "Solid" would be good if one had something more general, allowing one to try out different options such as use of Credentials. Just changing the name though does not give a more general protocol. To get that one would need to add some parameters and values to allow the client to know the difference. Without those features the change brings a semblance of generality without providing one. And one would need to have implementations make a change without a benefit to come with it.
+...: I think JSON folks may at an IRI - in a position where a short attributes would have done just as well - as something odd or even weird.
+Aaron: Those JWTs get sent with every authenticated request. Short claims like `iat` etc. allow more compact tokents. If we change it to IRI it takes more bytes.
+Pavlik: scopes have ofte IRI as values. Also we had proposal to exchange bigger token for short token used on each request, this would also go along UMA2.0 approach with delegating claims gathering.
+Aaron: Let's continue in issue.
+Henry: It's really up to those implementing this to decide. I have not implemented the OIDC connect protocol, so please take my statements with a grain of salt there.
```

Comment:
```suggestion
Aaron: we have some +1 on the issue. Pavlik suggested using IRIs for claim names and for scopes.
I generally don't see it in other systems but they can be used.

Pavlik: if we don't want to use IRI we would need to register that JWT claim

Aaron: is solid is taking route of registering strings with IANA, or using IRIs which we don't have to register?

Sarven: I don't know if we have global position on it, we did it for `acl` which has a history. `describedby` is registered by LDP spec.

Aaron: In that case, if something we will change to using IRI sounds like linked-data–friendly approach.

Sarven: If we think it can be used beyond solid, we could consider registering with IANA.

Henry: Moving from "WebID" to something more general such as "Solid" would be good if one had something more general, allowing one to try out different options such as use of Credentials. Just changing the name does not give a more general protocol. To get that, one would need to add some parameters and values to allow the client to know the difference. Without those features, the change brings a semblance of generality without providing one. And one would need to have implementations make a change without a benefit to come with it.
...: I think JSON folks may look at an IRI - in a position where a short attribute would have done just as well - as something odd or even weird.

Aaron: Those JWTs get sent with every authenticated request. Short claims like `iat`, etc., allow more compact tokens. If we change it to IRI it takes more bytes.

Pavlik: scopes often have IRI as values. Also we had proposal to exchange bigger token for short token used on each request; this would also go along UMA2.0 approach with delegating claims gathering.

Aaron: Let's continue in issue.

Henry: It's really up to those implementing this to decide. I have not implemented the OIDC connect protocol, so please take my statements with a grain of salt there.
```